tests/gc/panicoverride.nim

```diff
@@ -0,0 +1,14 @@
+
```

Comment:
this needs a comment, since it's only referenced in
lib/system/fatal.nim:18:25:  include "$projectpath/panicoverride"